docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -0,0 +1,205 @@
+|Text classification|multiclass classification|[GitHub issue data](https://github.com/dotnet/machinelearning-samples/blob/master/samples/csharp/end-to-end-apps/MulticlassClassification-GitHubLabeler/GitHubLabeler/Data/corefx-issues-train.tsv)|Area|Title, Description|
+
+## Train
+
+Once you select your scenario, data, and label, Model Builder trains the model.
+
+### What is training?
+
+Training is an automatic process by which Model Builder teaches your model how to answer questions for your scenario. Once trained, your model can make predictions with input data that it has not seen before. For example, if you are predicting house prices and a new house comes on the market, you can predict its sale price.
+
+Model Builder uses automated machine learning (AutoML), which automatically explores different machine learning algorithms and settings to find the one that best suits your scenario. Model Builder does not require any input or tuning from you. No prior expertise for ML is required.
+
+### How long should I train for?
+
+You can provide a training time. In general, training for a longer time produces a more accurate model. More training time is also required as the size of the training dataset increases. The following table gives some training time guidelines for datasets of increasing size.
+
+Dataset Size  | Dataset Type       | Avg. Time to train
+------------- | ------------------ | --------------
+0 - 10 Mb     | Numeric and Text   | 10 sec
+10 - 100 Mb   | Numeric and Text   | 10 min 
+100 - 500 Mb  | Numeric and Text   | 30 min 
+500 - 1 Gb    | Numeric and Text   | 60 min 
+1 Gb+         | Numeric and Text   | 3 hour+ 
+
+The exact time to train also depends on:
+
+- the type of columns that is, text vs numeric
+- the type of machine learning task (regression or classification)
+- the number of rows used for training
+- the number of feature columns used for training
+
+Model Builder has been tested for scale with a 1-TB dataset, but building a high-quality model for that size of dataset can take up to four days!
+
+## Evaluate
+
+Evaluation is the process of using the trained model to make predictions with new test data, and then measuring how good the predictions are.
+
+Model Builder splits the training data into a training set and a test set. The training data (80%) is used to train your model and the test data (20%) is held back to evaluate your model.  The metrics used for evaluation depend on the ML task. For more information, see [model evaluation metrics](resources/metrics.md).  
+
+### Sentiment analysis (binary classification)
+
+The default metric for binary classification problems is **accuracy**. Accuracy defines the proportion of correct predictions your model makes over the test dataset. The **closer to 100%, the better it is**. 
```

Comment:
I'd move all the default metrics to either the top or bottom of the each section for consistency and readability.